sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisExternalCatalogSuite.scala

```diff
@@ -57,6 +57,7 @@ class AnalysisExternalCatalogSuite extends AnalysisTest with Matchers {
     val plan = Project(Seq(func), testRelation)
     analyzer.execute(plan)
     verifyZeroInteractions(catalog)
+    catalog.dropTable("default", "t1", true, false)
```

Comment:
@sandeep-katta . This is a `inMemoryCatalog` created in this PR. The catalog will not affect the others.
Do you need to remove the location used for `default.t1`?

Reply:
yes need to delete the location

Reply:
In that case, this cannot guarantee what you want. This test can fail at line 59.
I'd like to recommend to you clean up the leftovers at `def afterAll`. Please refer the other test case which is overriding `afterAll` function. That will be the recommended and more robust way to do this.

Reply:
I will add in afterEach, table created in "query builtin functions don't call the external catalog" is affecting the testcase "check the existence of builtin functions don't call the external catalog".  As per SPARK-25464, database can only be created to empty location. So it is requried to delete the table created in every testcase